modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/VisibleSelectInput.scss

```diff
@@ -8,7 +8,7 @@
 }
 
 .lfr__ddm-form-field-type-select-arrow-down {
-	left: 95%;
+	left: 85%;
 	margin-left: auto;
 	position: absolute;
 }
```

Comment:
Hello, @Valmir-unicap, I found a misalignment using this percentage on Select fields on Builder. 
See:
![image](https://user-images.githubusercontent.com/26491796/223453853-166215cb-e11f-43f6-9581-80ec292ed5ca.png)

The best approach I see is to remove `left` and `position` properties.

Please, use:
```
.lfr__ddm-form-field-type-select-arrow-down {
	margin-left: auto;
}
```

I've tested with multiple selection to check if it would break something of this ticket ([LPS-167750](https://issues.liferay.com/browse/LPS-167750)) and it seems to be good. 

![image](https://user-images.githubusercontent.com/26491796/223455249-e2c6f3f5-28f4-45a0-aeb1-442c8e986618.png)

Reply:
Hello @alinedoleron , perfect thanks!